src/main/java/Duke.java

```diff
@@ -1,10 +1,186 @@
+     */
+    public static void add(ArrayList<Task> todoList,
+                           String taskDescription) {
+
+        Task task = new Task(taskDescription);
+        todoList.add(task);
+
+        System.out.println(TAB_SIZE + "You have added [" + taskDescription +
+                "] to your todo list");
+    }
+
+    /**
+     * lists down the items in the ArrayList and print
+     * them to the terminal
+     *
+     * @param todoList the ArrayLists that contains the stored items
+     */
+    public static void listItems(ArrayList<Task> todoList) {
+
+        if (todoList.size() == 0) {
+            System.out.println(TAB_SIZE + "Your todo list is current void of task," +
+                    " please add some tasks.");
+        } else {
+            System.out.println(TAB_SIZE + "TODO LIST:");
+        }
+
+        for (int i = 1; i <= todoList.size(); i++) {
+
+            System.out.println(TAB_SIZE + TAB_SIZE +
+                    i + "." + todoList.get(i-1).getStatusIcon() +
+                    todoList.get(i-1).getDescription());
+        }
+    }
+
+    /**
+     * marks the task number that user had completed
+     *
+     * @param todoList ArrayList that contains the tasks
+     * @param taskNumber task number that the user wants to mark as completed
+     */
+    public static void markTasks(ArrayList<Task> todoList, int taskNumber) {
+
+        todoList.get(taskNumber - 1).setDone(true);
+        System.out.println(TAB_SIZE + "Nice! I've marked this task as done:");
+        System.out.println(TAB_SIZE + " [✔] " +
+                todoList.get(taskNumber - 1).getDescription());
+    }
+
+    /**
+     * unmarks the task number from the todo list
+     *
+     * @param todoList ArrayList that contains the tasks
+     * @param taskNumber task number that the user wants to mark as incomplete
+     */
+    public static void unMarkTasks(ArrayList<Task> todoList, int taskNumber) {
+
+        todoList.get(taskNumber - 1).setDone(false);
+        System.out.println(TAB_SIZE + "Ok! I've marked this task as undone:");
+        System.out.println(TAB_SIZE + " [ ] " + todoList.get(taskNumber - 1).getDescription());
+    }
+
+
+    /**
+     * checks for potential errors in markTasks and unMarkTasks function
+     *
+     * @param command the command that the user inputs
+     * @param userSplitInputs String array that contains the split user command
+     * @param todoList Array List that contains the tasks
+     */
+    public static void errorCheckingForMarkTasks(String command, String[] userSplitInputs,
+                                                 ArrayList<Task> todoList) {
+
+        if (userSplitInputs.length == 1) {
+
+            System.out.println(TAB_SIZE+ "Please try again and include a task number.");
+            System.out.println(TAB_SIZE + "i.e., done 1");
+
+            return;
+        }
+
+        if (todoList.size() == 0) {
+            System.out.println(TAB_SIZE + "The todo list is currently empty. \n" +
```

Comment:
For multi line println I believe the successive lines need to be more indented to make it look more seamless